solution/ui/prototype/src/views/CacheExplorer.vue

```diff
@@ -3,23 +3,33 @@
         CACHE EXPLORER
         <ul>
             <li v-for="key in cacheKeys" v-bind:key="key">
-                {{key}}
-                <v-btn class="button" @click="deleteCacheItem(key)">Delete</v-btn>
+                {{ key }}
+                <v-btn class="button" @click="deleteCacheItem(key)"
+                    >Delete</v-btn
+                >
```

Comment:
All the changes in this file was from prettier formatting -- nothing new or changed.